src/exchange_adapters/base.ts

```diff
@@ -92,308 +92,309 @@ export interface ExchangeAdapterConfig {
 }
 
 export abstract class BaseExchangeAdapter {
-  protected readonly config: ExchangeAdapterConfig
```

Comment:
Can you revert the indentation changes on this file? It seems like you only added the `[ExternalCurrency.USDC, 'USDC'] line or was there something else?